CONTRIBUTING.md

```diff
@@ -126,6 +126,19 @@ branch that is checked out in a separate worktree.
     git log --stat -p aaaaaa...bbbbbb
     ```
 
+## Polish a change
+
+Prior to pushing your change, remember to run clang-format on the `c/cpp` files, clang-format workflow may fail otherwise. You can do so by running the following command.
+
+`find . -regex '.*\.\(c\|cpp\|h\|hpp\)' | sort | xargs clang-format --verbose -i`.
+
+To do this, you will need clang version 13.0.0, which you can download from [llvm](https://github.com/llvm/llvm-project/releases/tag/llvmorg-13.0.0). Then add its bin directory to PATH.
+
+Note: If you are using Fedora, CentOS, or SUSE distribution, you can install `clang+llvm-13.0.0-x86_64-linux-gnu-ubuntu-xx.xx.tar.xz`.
+
+If you wish to format a committed or staged file. You can use clang-format git integration: [`git-clang-format`](https://github.com/llvm/llvm-project/blob/9e634b35ff51d0eb2b38013111491e88bdbae388/clang/tools/clang-format/git-clang-format).
```

Comment:
Use a comma: `If you wish to format a committed or staged file, you can use`

Reply:
What is the main benefit of using `git clang-format` over running `clang-format` manually?

Reply:
The main benefit of `git clang-format` is the ability to format a specific commit without having to format, and fixup commits.
Or it can be used on only the staged files.

Reply:
Thanks Sherry. In terms of practical workflows, is `git clang-format` always preferable over manually running `clang-format`? Is there a reason not to recommend `git clang-format`? (Sorry for asking these simple questions; I use the [vim-clang-format](https://github.com/rhysd/vim-clang-format) plugin for automatic formatting.)

Reply:
I think it really depend on what situation the developer is in. I personally would like to solely rely on that single line command / or some sort of plugins of IDE. However if a developer accidentally committed a change without formating (maybe because forgot to formating) and wish to do it after in one line, then git-clang-format is the way to go. Further, if we need to import an upstream project in the future, and that project is not completely formated, then some sort of surgical formatting like git-clang-format will meet our need (to avoid noise in PR).